Langs/README.md

```diff
@@ -0,0 +1,38 @@
+# Guidelines
+
+## Requirements
+
+- **Find Your Language Code Here:** https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm
```

Comment:
I don't like the bold text here. If you really wanted to use markdown then you could do something like this:
`Find your language code [here](https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm)`

Reply:
Removed the bold text and placed a embedded hyperlink.